helpers.js

```diff
@@ -0,0 +1,55 @@
+const log4js  = require('log4js');
+const LOG     = log4js.getLogger('helpers');
+const format  = require('format');
+const _       = require('lodash');
+
+// Using getMessages will automatically use already cached messages when available,
+//     and will fetch more messages only if needed. This way we don't have to rely
+//     on the promise function structure every time we want to retrieve a message
+exports.getMessages = async function getMessages(message) {
```

Comment:
if we start using this in handlers, how long will it take a new message to enter the cache?

is this just a way for handlers that need messages for sampling to get them quickly without a network request? i guess those kinds don't need the _latest_ messages ASAP.

Reply:
When the client starts, there are no messages in the cache, which is why when we looked earlier, it seemed like the 'messages' data object only had a single message. After the client starts, channel messages will fill that cache automatically, and running `fetchMessages()` backfills it. Using this method, we will only need to run `fetchMessages` once per instance to backfill the memory, and all future messages are automatically loaded into the cache memory.